src/node_http2.cc

```diff
@@ -204,7 +203,7 @@ Http2Options::Http2Options(Http2State* http2_state, SessionType type) {
       uint32_t val = buffer[IDX_SETTINGS_ ## name];                            \
       entries[count++] =                                                       \
           nghttp2_settings_entry {NGHTTP2_SETTINGS_ ## name, val};             \
-    } } while(0)
+    } } while (0)
```

Comment:
Is your plan to squash these all into one commit? Otherwise it would be good if every commit passed linting on its own

Reply:
Yes, once they're all reviewed.